components/Directions.js

```diff
@@ -0,0 +1,187 @@
+import React, { useEffect, useRef } from "react";
+import { StyleSheet, AsyncStorage, TouchableOpacity } from "react-native";
+import MapView, { PROVIDER_GOOGLE, Polyline } from "react-native-maps";
+import { View, Button, Text, Icon } from "native-base";
+import PolyLine from "@mapbox/polyline";
+import { BottomMenu } from "./BottomMenu";
+import PropTypes from "prop-types";
+
+
+const mapPosition = {
+    sgwCoord: {
+        latitude: 45.496557,
+        longitude: -73.578896,
+        latitudeDelta: 0.005,
+        longitudeDelta: 0.005,
+    },
+    loyCoord: {
+        latitude: 45.457841,
+        longitude: -73.640307,
+        latitudeDelta: 0.01,
+        longitudeDelta: 0.01,
+    }
+};
+
+/**
+ * US1 - As a user, I would like to navigate through SGW campus.
+ * US2 - As a user, I would like to navigate through Loyola campus.
+ * 
+ * This is our main screen which includes all the components inside a map.
+ */
+function Directions (props) {
+
+    const [decodedPolylines, setDecodedPolylines] = React.useState([]);
+    const [detailedInstructions, setDetailedInstructions] = React.useState();
+    const [directionRegion, setDirectionRegion] = React.useState(props.initialRegion);
+    const mapRef = useRef(null);
+
+    const handleMapPress = () => {
+        const region = {
+            latitude: 37.78825,
+            longitude: -122.4324,
+            latitudeDelta: 0.0922,
+            longitudeDelta: 0.0421
+        };
+        mapRef.current.animateToRegion(region, 1000);
+    };
+
+    const goBackPressHandler = () => {
+        props.navigation.goBack();
+    };
+
+    // useEffect(() => {
```

Comment:
why is this commented?

Reply:
Because the Directions.js file is not completed, as it is being worked on in the **feature/outdoor-directions** by @AlainJobs